tests/Billing/Entities/PreferenceTest.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace ArrowSphere\PublicApiClient\Tests\Billing\Entities;
+
+use ArrowSphere\PublicApiClient\Billing\Entities\Preference;
+use ArrowSphere\PublicApiClient\Exception\EntityValidationException;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * Class PreferenceTest
+ */
+class PreferenceTest extends TestCase
```

Comment:
There's a `AbstractEntityTest` class to standardize the testing of entities
But it's almost the same